### PR TITLE
Add filter editable playlist

### PR DESCRIPTION
**Pull request by Arfaz7**

### CODE REVIEW

File: src/sub-clients/collections/client/CollectionsClient.test.pact.ts [hunk 1]
```diff
@@ -248,6 +251,7 @@ describe('CollectionsClient', () => {
             page,
             size,
             origin,
+            canEdit: true,
```
Comment (rlewan) [inline src/sub-clients/collections/client/CollectionsClient.test.pact.ts:254]: I'm not sure if I understand the new filter correctly, but I interpret this as "get me all the collections that I can edit, so I either own them or the owner set them to be editable by anyone".

In that case, based on the setup we have with the view only permission, shouldn't the call return a single collection only?

Comment (rlewan) [reply src/sub-clients/collections/client/CollectionsClient.test.pact.ts:254]: I would also consider adding a separate test and interaction that test around the new permissions property and the new filter.

Comment (Arfaz7) [reply src/sub-clients/collections/client/CollectionsClient.test.pact.ts:254]: Thanks Rafal, you understand it correctly.
I'll add a test for that :)

**Review by rlewan — APPROVED** [review]
Looks good, thanks! 👌

**Pull request closed without merging**